src/components/Passport.js

```diff
@@ -142,7 +142,7 @@ Passport.Mini = function MiniPassport({ point, className, inverted, ...rest }) {
           patp={name}
           size={64}
           margin={0}
-          colorway={inverted ? INVERTED_COLORWAY : undefined}
+          colors={inverted ? INVERTED_COLORWAY : undefined}
```

Comment:
was this not fixed in the sigil branch?

Reply:
The branch this PR is coming from contains mino, which contains the sigil branch, so no. I overlooked this.